### DISCUSSION TITLE
Swagger with hosted files does not work after upgrade

### DISCUSSION THREAD

**Discussion by zemalex** [fastapi/fastapi#10426] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import APIRouter
from fastapi.openapi.docs import get_swagger_ui_html, get_swagger_ui_oauth2_redirect_html, get_redoc_html


class FastApiSwagger:

    def __init__(self, app):
        self.router = APIRouter()
        self._app = app
        self.router.add_api_route("/docs", self.custom_swagger_ui_html, methods=["GET"], include_in_schema=False)
        self.router.add_api_route("/docs/oauth2-redirect", self.swagger_ui_redirect, methods=["GET"], include_in_schema=False)
        self.router.add_api_route("/redoc", self.swagger_ui_redirect, methods=["GET"], include_in_schema=False)

    async def custom_swagger_ui_html(self):
        """
        Static swagger method
        """
        return get_swagger_ui_html(
            openapi_url=self._app.openapi_url,
            title="Verdi365" + " - Swagger UI",
            oauth2_redirect_url=self._app.swagger_ui_oauth2_redirect_url,
            swagger_js_url="/static/swagger-ui-bundle.js",
            swagger_css_url="/static/swagger-ui.css",
    )

    @staticmethod
    async def swagger_ui_redirect():
        """
        Static swagger method
        """
        return get_swagger_ui_oauth2_redirect_html()

    async def redoc_html(self):
        """
        Static swagger method
        """
        return get_redoc_html(
            openapi_url=self._app.openapi_url,
            title="Verdi365" + " - ReDoc",
            redoc_js_url="/static/redoc.standalone.js",
        )
```

### Description
The handle of static files are done in main method with app, 
I've checked with host/static/<file_name> and I can get the static file, so they are handled properly
The swagger code is in the router in order to split my big app to small routers.

after an update of the fastAPI to the latest version, 
the swagger with self hosted files stoped to work, 
it raises the next errors in the web console:
1. Uncaught SyntaxError: expected expression, got '='
2. Uncaught ReferenceError: SwaggerUIBundle is not defined
    <anonymous> https://10.197.210.88:9003/docs#:15
   I've tried to follow the FastAPI tutorial and still same error, 
   
   while line 15 is:
   const ui = SwaggerUIBundle({{
  

### Operating System

Linux

### FastAPI Version

0.103.2

### Pydantic Version

2.4.2

### Python Version

3.8

**Comment by jhanseth** [comment]
Anyone find a fix for this? :)
